### PR TITLE
Reworks GBP values to be lower impact, adds GBP impact to visual and audio changes, removes GBP impact from unrelated tags

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request
Fixes GBP values to be sane. Like, jesus, who wrote these numbers?
## Why It's Good For The Game
N/A

### CODE REVIEW

File: .github/gbp.toml [hunk 1]
```diff
@@ -2,19 +2,16 @@ no_balance_label = "GBP: No Update"
 reset_label = "GBP: Reset"
 
 [points]
-"Atomic" = 2
```
Comment (Kylerace) [inline .github/gbp.toml:5]: why get rid of atomic?

Comment (Iamgoofball) [reply .github/gbp.toml:5]: We don't use it.

Comment (LemonInTheDark) [reply .github/gbp.toml:5]: we do use it

File: .github/gbp.toml [hunk 2]
```diff
@@ -2,19 +2,16 @@ no_balance_label = "GBP: No Update"
...
-"Sound" = 1
-"Sprites" = 3
-"Unit Tests" = 6
+"Performance" = 2
+"Priority: CRITICAL" = 4
+"Priority: High" = 3
+"Quality of Life" = -1
+"Refactor" = 2
+"Sound" = -1
+"Sprites" = -1
```
Comment (LemonInTheDark) [inline .github/gbp.toml:16]: I want more, not less of these

File: .github/gbp.toml (hunk 1 above)
Comment (Iamgoofball) [reply .github/gbp.toml:5]: we used it in 34 PRs out of 38,000 PRs

no, we don't use it

File: .github/gbp.toml [hunk 3]
```diff
@@ -2,19 +2,16 @@ no_balance_label = "GBP: No Update"
...
 [points]
-"Atomic" = 2
-"Balance/Rebalance" = -8
+"Balance/Rebalance" = -1
 "Code Improvement" = 2
-"Feature" = -10
-"Feedback" = 2
-"Fix" = 3
+"Feature" = -1
+"Fix" = 1
```
Comment (LemonInTheDark) [inline .github/gbp.toml:8]: 1 fix is nowhere near worth one feature

File: .github/gbp.toml [hunk 4]
```diff
@@ -2,19 +2,16 @@ no_balance_label = "GBP: No Update"
...
-"Priority: High" = 15
-"Quality of Life" = 1
-"Refactor" = 10
-"Sound" = 1
-"Sprites" = 3
-"Unit Tests" = 6
+"Performance" = 2
+"Priority: CRITICAL" = 4
+"Priority: High" = 3
+"Quality of Life" = -1
```
Comment (LemonInTheDark) [inline .github/gbp.toml:13]: what?

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

File: .github/gbp.toml (hunk 1 above)
Comment (LemonInTheDark) [reply .github/gbp.toml:5]: it's a new damn label

Comment (Mothblocks) [reply .github/gbp.toml:5]: do u know how long its existed

**Pull request closed without merging**